### PR TITLE
GSOC Template Script.

### PR DESCRIPTION
**Pull request by dialmformart**
First GSOC template script on matlab.

### CODE REVIEW

File: matlabCode/gsoc/gsoc_tenorio_2018 [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+% MTenorio, 2018
+
+%%
+clear all
+root_dir = 'C:\Users\Marcus\Documents'; %Set manually your root directory
+project_label = 'templates';
+
+
+events_tsv_name = fullfile(root_dir,project_label,...
+    '_events.tsv'); %make sure you create a new blank file named '_events.tsv' on tamplates directory
```
Comment (dorahermes) [inline matlabCode/gsoc/gsoc_tenorio_2018]: This file should go with appropriate naming into:
sub-01/ses-01/func/sub-01_ses-01_task-FullExample_run-01_events.tsv

File: matlabCode/gsoc/_events.tsv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+onset	duration	trial_type	response_time
+sub-01	0	go	0
```
Comment (dorahermes) [inline matlabCode/gsoc/_events.tsv]: Onsets should be stated in seconds

File: matlabCode/gsoc/gsoc_tenorio_2018 (hunk 1 above)
Comment (dorahermes) [reply matlabCode/gsoc/gsoc_tenorio_2018]: This file should have names consistent with other Matlab templates.

**Comment by Remi-Gau** [comment]
Will close this as this has gone dormant.

**Pull request closed without merging**